src/cli/export/export.go

```diff
@@ -0,0 +1,41 @@
+package export
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var exportCommands = []func(cmd *cobra.Command) *cobra.Command{
+	addOneDriveCommands,
+	// addExchangeCommands,
+	// addSharePointCommands,
```

Comment:
TODOs here, please.  Otherwise we're not clear why these values are commented out.